templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,11 @@ below features are supported.
 
 ## Changes in Zulip 2.2
 
+**Feature level 8**
+* [`GET /users`](/api/get-all-users), [`GET /users/{user_id}`](/api/get-user)
+  and [`GET /users/me`](/api/get-profile): User objects now contain the
+  `is_owner` field as well.
+
```

Comment:
Your original version of this commit added lines to "Feature level 7".  You should never add new changes to the documentation for an existing feature level already declared in `version.py`.  Except in rare cases (like what we did with 7), each API change should get its own feature level entry.

Reply:
Okay, I will keep this in mind from next time.